clients/fides-js/src/lib/cookie.ts

```diff
@@ -217,3 +217,31 @@ export const saveFidesCookie = (cookie: FidesCookie) => {
     CODEC
   );
 };
+
+export const setConsentCookieAcceptAll = (defaults: CookieKeyConsent): void => {
+  if (defaults === undefined) {
+    // eslint-disable-next-line no-console
+    console.error("Unable to set consent cookie to accept all: invalid defaults.");
+    return;
+  }
+
+  // Override all consent values to true and save the cookie
+  const entries: [string, boolean][] = Object.keys(defaults).map((key) => [key, true]);
+  const cookieKeyConsent = Object.fromEntries(entries);
+  const cookie: FidesCookie = getOrMakeFidesCookie(cookieKeyConsent);
```

Comment:
actually, do we want to use this function? I think this one queries the existing cookie, and then merges it with the passed in `defaults`, but would prioritize the existing cookie. for example, when I open up http://localhost:3000/fides-js-demo.html, I already have a cookie with 

```
{
  "data_sales": true,
  "tracking": false,
  "analytics": false
}
```

and when I click "Accept all", that cookie is still the same since the `cookieKeyConsent` where every value is `true` doesn't actually overwrite anything

Reply:
I haven't thought much about cookie updating, and you're right- this definitely doesn't work right now. We could modify `getOrMakeFidesCookie` so that it works for this use case, I'm open to ideas here

Reply:
I think we might want to `getOrMakeFidesCookie` and then overwrite its `consent`. I'm not 100% sure about this, but let me try it out and I'll push a commit and you can tell me if it makes sense

Reply:
https://github.com/ethyca/fides/pull/3191/commits/5f649968a3498d3adf52d24fac4a27f636179553

Reply:
you can see this is working since you can "Accept all", then refresh the page and see the `Consent JSON` values update to all be true, then "Reject all" and see them all change to false